site/docs/tutorial/ios-app.md

```diff
@@ -174,7 +186,6 @@ objc_library(
          "UrlGet/main.m",
     ],
     hdrs = glob(["UrlGet/*.h"]),
-    xibs = ["UrlGet/UrlGetViewController.xib"],
```

Comment:
I believe this should instead be in the `data` attribute, see [here](https://github.com/bazelbuild/rules_apple/blob/master/doc/resources.md#migration).

Reply:
Thank you for your comment. I agree with you! I will update here following your advice.